src/cozy/react/components/AvailablePlatforms.jsx

```diff
@@ -30,15 +30,23 @@ export const InstallNativeAppButton = props => {
     setSmartphoneModalOpened(false)
   }, [setSmartphoneModalOpened])
 
+  const onClick = () => {    
+    if(mobileOS === null) {
+      handleOpenModal()
+    } else {
+      props.onClick?.()
+      window.location.href = storeLinksPerOS[mobileOS].storeUrl
```

Comment:
@Ldoppea is it ok to use window.location.href now? I'm sure I asked you this question not long ago

Reply:
Today on Flagship app we have the following behavior:
- `window.open()` is intercepted and triggers the IAB if it is an external URL (external = different host than the cozy fqdn)
- `window.location.href` is intercepted and triggers the IAB if it is an external URL
- custom schemes like `tel:`, `mail:` etc should be intercepted and opened with the OS Linking method ([cf](https://github.com/cozy/cozy-flagship-app/pull/213))
- custom `market:` schemes has no rule, so I suppose it will be detected as external URL and opened in IAB (or maybe it will be intercepted on the webview native part)
  - In that scenario I cannot predict what will happen on different OS based on my current knowledge
  - But as @zatteo did the PR I suppose this worked on Android, and as iOS uses a classic URL it should be OK

Reply:
So now, there is no more restriction to force the use of the AppLinker to open an app in fact. Right?

Reply:
> So now, there is no more restriction to force the use of the AppLinker to open an app in fact. Right?

Yes, no restriction. I still prefer the AppLinker version as it is more predictable (no need to analyse the url), but today the URL interception seems stable enough